### PR TITLE
update user-account-management.md

### PR DESCRIPTION
**Pull request by tiancaiamao**
### Which TiDB version(s) do your changes apply to? (Required)

- [x] master (the latest development version)
- [x] v4.0 (TiDB 4.0 versions)
- [ ] v3.1 (TiDB 3.1 versions)
- [ ] v3.0 (TiDB 3.0 versions)
- [ ] v2.1 (TiDB 2.1 versions)

### What is the related PR or file link(s)?

Update user-account-management.md

- This PR is translated from:
- Other reference link(s):

### CODE REVIEW

File: user-account-management.md [hunk 1]
```diff
@@ -93,7 +82,9 @@ CREATE USER 'test';
 CREATE USER 'test'@'%' IDENTIFIED BY '';
 ```
 
-下面的例子用 `CREATE USER` 和 `GRANT` 语句创建了四个账户：
+为一个不存在的用户授权，是否会自动创建用户，行为受到 `sql_mode` 影响。如果 `sql_mode` 中包含 `NO_AUTO_CREATE_USER`，则 `GRANT` 不会自动创建用户并报错。
```
Comment (lilin90) [inline user-account-management.md:85]:
```suggestion
为一个不存在的用户授权时，是否会自动创建用户的行为受 `sql_mode` 影响。如果 `sql_mode` 中包含 `NO_AUTO_CREATE_USER`，则 `GRANT` 不会自动创建用户并报错。
```

File: user-account-management.md [hunk 2]
```diff
@@ -212,17 +203,22 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
-2. 然后使用 `root` 登录后修改密码：
+2. 使用修改之后的配置启动 TiDB，然后使用 `root` 登录后修改密码：
 
     {{< copyable "shell-regular" >}}
 
     ```bash
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
 
+设置 `skip-grant-table` 之后，启动 TiDB 进程会增加检查，只有操作系统的 `root` 用户才能执行。
```
Comment (lilin90) [inline user-account-management.md:214]: 稍有点儿困惑，最后的`才能执行`是指执行什么呢？建议补全一下~

File: user-account-management.md [hunk 3]
```diff
@@ -212,17 +203,22 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+
 ## `FLUSH PRIVILEGES`
 
-如果授权表已被直接修改，运行如下命令可使改动立即生效：
+用户以及权限相关的信息都是存储在 TiKV 服务器中的，TiDB 在进程内部会缓存。一般通过 `CREATE USER`，`GRANT` 等语句的修改，都能够在整个集群迅速生效。
```
Comment (lilin90) [inline user-account-management.md:218]:
```suggestion
用户以及权限相关的信息都存储在 TiKV 服务器中，TiDB 在进程内部会缓存这些信息。一般通过 `CREATE USER`，`GRANT` 等语句来修改相关信息时，可在整个集群迅速生效。
```

File: user-account-management.md [hunk 4]
```diff
@@ -212,17 +203,22 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
+如果遇到网络或者其它因素影响，由于 TiDB 会周期性的更新缓存信息，正常情况最多 15 分钟左右生效。
```
Comment (lilin90) [inline user-account-management.md:219]:
```suggestion
如果遇到网络或者其它因素影响，由于 TiDB 会周期性地更新缓存信息，正常情况下，最多 15 分钟左右生效。
```

**Comment by imtbkcat** [comment]
ping @tiancaiamao

File: user-account-management.md (hunk 2 above)
Comment (jackysp) [reply user-account-management.md:214]: 执行启动

Comment (jackysp) [inline user-account-management.md:214]:
```suggestion
设置 `skip-grant-table` 之后，启动 TiDB 进程会增加操作系统用户检查，只有操作系统的 `root` 用户才能启动 TiDB 进程。
```

**Comment by jackysp** [comment]
PTAL @lilin90

File: user-account-management.md [hunk 5]
```diff
@@ -212,17 +203,22 @@ TiDB 将密码存在 `mysql.user` 系统数据库里面。只有拥有 `CREATE U
...
     mysql -h 127.0.0.1 -P 4000 -u root
     ```
 
+设置 `skip-grant-table` 之后，启动 TiDB 进程会增加操作系统用户检查，只有操作系统的 `root` 用户才能启动 TiDB 进程。
+
 ## `FLUSH PRIVILEGES`
 
-如果授权表已被直接修改，运行如下命令可使改动立即生效：
+用户以及权限相关的信息都存储在 TiKV 服务器中，TiDB 在进程内部会缓存这些信息。一般通过 `CREATE USER`，`GRANT` 等语句来修改相关信息时，可在整个集群迅速生效。
+如果遇到网络或者其它因素影响，由于 TiDB 会周期性地更新缓存信息，正常情况下，最多 15 分钟左右生效。
```
Comment (lilin90) [inline user-account-management.md:219]:
```suggestion
用户以及权限相关的信息都存储在 TiKV 服务器中，TiDB 在进程内部会缓存这些信息。一般通过 `CREATE USER`，`GRANT` 等语句来修改相关信息时，可在整个集群迅速生效。如果遇到网络或者其它因素影响，由于 TiDB 会周期性地更新缓存信息，正常情况下，最多 15 分钟左右生效。
```

**Pull request merged**